flowmachine/flowmachine/features/location/redacted_labelled_spatial_aggregate.py

```diff
@@ -0,0 +1,62 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import List
+
+from flowmachine.features.location.labelled_spatial_aggregate import (
+    LabelledSpatialAggregate,
+)
+from flowmachine.core import Query
+from flowmachine.core.mixins import GeoDataMixin
+
+
+class RedactedLabelledSpatialAggregate(GeoDataMixin, Query):
+    """
+    Query that drops any locations that, when disaggregated by label, reveal a number of subscribers
+    less than redaction_threshold
+    Parameters
+    ----------
+    labelled_spatial_aggregate: LabelledSpatialAggregate
+        The LabelledSpatialAggregate query to redact
+    redaction_threshold: int default 15
+        If any labels within a location reveal less than this number of subscribers, that location is dropeed
+    """
+
+    def __init__(
+        self,
+        *,
+        labelled_spatial_aggregate: LabelledSpatialAggregate,
+        redaction_threshold: int = 15,
+    ):
+
```

Comment:
You'll need `self.spatial_unit = labelled_spatial_aggregate.spatial_unit` here as well, for the get-result-as-geojson stuff to work.